Services/ZenDesk/api.ts

```diff
@@ -50,7 +53,7 @@ app.post(
   zValidator(
     'form',
     z.object({
-      name: z.string().min(2).optional(),
+      name: z.string().min(2),
```

Comment:
You seem to have overridden the base decision. As "name" is now not required, this should be set as optional.

```suggestion
      name: z.string().min(2).optional(),
```